scaffolding-chef-infra/tests/user-linux/habitat/plan.sh

```diff
@@ -0,0 +1,8 @@
+pkg_name=user-linux
+pkg_origin=chef
+pkg_version="1.0.0"
+pkg_upstream_url="http://chef.io"
+pkg_scaffolding="ci/scaffolding-chef-infra"
+pkg_svc_user=("root")
+scaffold_policy_name="ci"
+scaffold_chef_client="chef/chef-client/14.13.11"
```

Comment:
Any reason to pin here? I'd like to run latest so we can catch issues if/when they happen.

Reply:
Same here https://github.com/chef/effortless/pull/100/files#diff-2ef69c17f3405c345463fc122527ec09R13

Reply:
Pinning here is part of the test. We need to test that pinning works.

This user plan is a test harness, so it's ok to pin.

I'll add a comment that outlines this.

Reply:
Okay so if I understand then we would also have a test that doesn't have the client pinned if we wanted to test the latest clients?